portal/patient/templates/DefaultHome.tpl.php

```diff
@@ -31,7 +31,7 @@
     </div>
 </div>
 <div class="container">
-    <div class='jumbletron jumbletron-sm'>
+    <div class='jumbotron jumbotron-fluid'>
     <div class="jumbotron">
```

Comment:
I'll fix this issue right here.

Reply:
originally wanted a jumbotron-sm. Where can I work with best bet at staying out of your way due to us merging in your branch? Is your next commit going to include many files?